client-v2/src/services/editionsApi.ts

```diff
@@ -80,6 +80,14 @@ export const publishIssue = async (id: string): Promise<void> => {
   }).then(() => {});
 };
 
+export const checkIssue = async (id: string): Promise<string[]> => {
+  return pandaFetch(`/editions-api/issues/${id}/preflightchecks`, {
+    method: 'GET'
```

Comment:
`/editions-api/issues/${id}/preflightchecks`
to
`/editions-api/issues/${id}/preflight-checks`

Reply:
Done.